accumulo_plugin/plugin-accumulo-2.0/PelzCryptoService.java

```diff
@@ -121,6 +142,12 @@ public FileDecrypter getFileDecrypter(CryptoEnvironment environment) {
       case AESGCMCryptoModule.VERSION:
         cm = new AESGCMCryptoModule(this.keyLocation, this.keyManager);
         return (cm.getDecrypter(fek));
+      case AES_CBC_CRYPTO_VERSION:
```

Comment:
Because you're able to use the PelzCryptoService's crypto modules, you can bundle these under the same case:  

    case AESGCMCryptoModule.VERSION:
    case AES_CBC_CRYPTO_VERSION:
        cm = new AESGCMCryptoModule(this.keyLocation, this.keyManager);
        return (cm.getDecrypter(fek));